src/main/java/stringAddCalculator/Divider.java

```diff
@@ -0,0 +1,29 @@
+package stringAddCalculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Divider {
+    private final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    private ArrayList<String> DELIMITER = new ArrayList<>(Arrays.asList(",", ":"));
+
+    List<String> getSplitByCustomDelimiter(String input) {
+        Matcher matcher = Pattern.compile(CUSTOM_DELIMITER).matcher(input);
+        if(matcher.find()){
+            return Arrays.asList(matcher.group(1),matcher.group(2));
```

Comment:
```suggestion
            return Arrays.asList(matcher.group(1), matcher.group(2));
```
매직넘버가 있네요
1과 2가 뜻하는게 무엇일까요?